src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,55 +1,132 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { getData } from '../../api/api';
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+
+interface Props {
+  addMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = (props) => {
+  const { addMovie } = props;
+  const [searchMovie, setSearchMovie] = useState('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
```
searchMovie sounds like some method